virttest/qemu_devices/qdevices.py

```diff
@@ -1051,7 +1051,8 @@ def __init__(self, backend, params=None):
     def get_children(self):
         """ Device bus should be removed too """
         devices = super(QObject, self).get_children()
-        if self.hook_drive_bus:
+        if (hasattr(self, 'hook_drive_bus') and
+                self.hook_drive_bus):
```

Comment:
For shortening, it can be `if getattr(self, 'hook_drive_bus', None):`

Reply:
Change done.